lib/message.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'send-sms'
+
+class Message
+  def initialize(sms = SendText.new)
```

Comment:
Try using environmental variables to store your details so you can upload full code